ingestion/create_cadet_databases_source/source.py

```diff
@@ -201,6 +219,21 @@ def create_table_domain_mcps(
                 aspect=DomainsClass(domains=[domain_urn]),
             )
             table_domain_mcps.append(mcp)
+
+            database_container_key = mcp_builder.DatabaseKey(
+                database=database,
+                platform=PLATFORM,
+                instance=INSTANCE,
+                env=ENV,
+                backcompat_env_as_instance=True,
+            )
+            db_urn = database_container_key.as_urn()
+            table_domain_mcps.append(
+                MetadataChangeProposalWrapper(
+                    entityUrn=dataset_urn,
+                    aspect=ContainerClass(container=db_urn),
+                )
+            )
```

Comment:
we should remove this part i reckon, as it didn't work to recreate the `IsPartOf` relationship